helm/README.md

```diff
@@ -63,9 +63,9 @@ The following example uses meta/llama-3-8b-instruct with an existing secret as t
 If you specify secrets as shown in this example, you cannot set the API key directly in the file or on the CLI. Instead, first create the secrets, as shown in the following example:
 
 ```bash
-kubectl -n nim create secret docker-registry registry-secret --docker-server=nvcr.io --docker-username='$oauthtoken' --docker-password=$NGC_CLI_API_KEY
+kubectl -n nim create secret docker-registry registry-secret --docker-server=nvcr.io --docker-username='$oauthtoken' --docker-password=$NGC_API_KEY
 
-kubectl -n nim create secret generic ngc-api --from-literal=NGC_CLI_API_KEY=$NGC_CLI_API_KEY
+kubectl -n nim create secret generic ngc-api --from-literal=NGC_API_KEY=$NGC_API_KEY
```

Comment:
This is incorrect for this version of the chart. The first value must be NGC_CLI_API_KEY. The second can be NGC_API_KEY

Reply:
Thank you!